apps/api-express/src/app/daos/search.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+import { runScholarsScraper } from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+
+export async function findQueryResults(
+  query: string,
+  opts?: {
+    numberOfArticles?: number;
+    maxNumberOfParagraphs?: number;
+  }
+): Promise<ParsedArticleParagraphStandalone[]> {
+  // The query is the impact and the recommendation is left blank
```

Comment:
Can you clarify this comment?  Not sure I understand it

Reply:
Oh the comment is now changed to match with passing query in twice